src/main/java/org/killbill/billing/plugin/analytics/dao/factory/BusinessFactoryBase.java

```diff
@@ -418,14 +418,23 @@ protected Long getInvoicePaymentRecordId(final UUID invoicePaymentId, final Tena
                     error = e1;
                 }
             }
+        } catch (final RuntimeException e) {
```

Comment:
What do you have in mind with catching this `RuntimeException` - are we trying to handle bug related exception or if not what is the scenario?